packages/ui/src/working-groups/queries/workingGroups.graphql

```diff
@@ -157,8 +157,8 @@ fragment WorkingGroupOpeningFieldsConnection on WorkingGroupOpeningConnection {
   }
 }
 
-query getWorkingGroupOpeningsConnection($groupId_eq: ID, $first: Int, $last: Int) {
-  workingGroupOpeningsConnection(where: { group_eq: $groupId_eq }, first: $first, last: $last) {
+query getWorkingGroupOpeningsConnection($groupId_eq: ID, $status_json: JSONObject, $first: Int, $last: Int) {
+  workingGroupOpeningsConnection(where: { group_eq: $groupId_eq, status_json: $status_json }, first: $first, last: $last) {
     ... WorkingGroupOpeningFieldsConnection
   }
```

Comment:
Oh, one last thing – I've forgot to check this. I think that it might be better to have a dedicated query:

```
query CountWorkingGroupOpenings($groupId_eq: ID, $status_json: JSONObject, $first: Int, $last: Int) {
  workingGroupOpeningsConnection(where: { group_eq: $groupId_eq, status_json: $status_json }, first: $first, last: $last) {
    id
  }
```

This way the GraphQL server will do a simple select to return the count. The `...WorkingGroupOpeningFields` fetches too much data and it might be slow (joins, etc).